garnett/trajectory.py

```diff
@@ -375,23 +359,79 @@ def __ne__(self, other):
                             details="Use to_hoomd_snapshot with no argument.")
     def make_snapshot(self):
         "Create a HOOMD-blue snapshot object from this frame."
-        self.load()
```

Comment:
Not clear to me why remove the `load` statement.

Reply:
This is called by `self.to_hoomd_snapshot` so it's redundant to have it twice.